README.md

```diff
@@ -114,19 +114,19 @@ I highly recommend uninstalling the game through Steam, deleting the game folder
 After you have done everything above and got a clean SSE installation ready, start the Launcher and and let it do the initial graphics check. Do not worry about this part as the installation will replace this graphics settings.
 Start the game and exit once you're in the main menu.
 
-### Downgrade Skyrim Anniversary Edition to Special Edition
+#### Downgrade Skyrim Anniversary Edition to Special Edition
```

Comment:
The heading level here and the table of contents disagree on what level this heading should be. I figure this should be under "Installation".